templates/server/src/main/resources/archetype-resources/core/src/test/java/__packageInPathFormat__/general/common/base/ApplicationTest.java

```diff
@@ -0,0 +1,19 @@
+package ${package}.general.common.base;
+
+import org.junit.jupiter.api.Test;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.boot.test.context.SpringBootTest.WebEnvironment;
+
+import com.devonfw.module.test.common.base.ModuleTest;
+import com.example.application.sampleapp.SpringBootApp;
+
+/**
+ * This test verifies that {@link SpringBootApp} is able to startup.
+ */
+public class ApplicationTest extends ApplicationComponentTest {
+  @Test
+  public void contextLoads() {
+    // just ensure the app can startup
+    SpringBootApp.main(new String[] {});
```

Comment:
As stated before, this line should be removed. JUnits should never call `main` methods as these may call `System.exit` aborting the entire JVM running all tests. Besides this currently starts the app twice - once via springboot-test and again via main method.
Instead add some assertion here e.g. test that some injection works as expected.